spec/mse_spec_helper.rb

```diff
@@ -1,5 +1,18 @@
 # frozen_string_literal: true
 
+# -- Changes that directly affect RSpec -- #
+module RSpec
```

Comment:
Here is fine for now — let's remember to move this to [custom-matchers](https://github.com/umts/custom-matchers).